process.py

```diff
@@ -190,7 +190,7 @@ def remove_groups(api, users, groups_to_remove_from):
         {
             "paging": False,
             "filter": "name:in:[%s]" % ",".join(groups_to_remove_from),
-            "fields": ("id,name,users"),
+            "fields": ("*"),
```

Comment:
`fields=*` should work just fine, but just for the record: when updating an API object (GET+modify+PUT) we typically use `fields=:owner` (which is the default if no fields param is requested).

https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-235/web-api.html#webapi_metadata_field_filter

> owner | Returns all persisted  property on an object where the object is the owner of all properties,  this payload can be used to update through the API.